examples/metrics/exemplar/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Do not include this file if empty. We have it in a lot of examples, but I think it is confusing.

Reply:
Fixed.